.github/workflows/lighthouse.yml

```diff
@@ -0,0 +1,18 @@
+name: lighthouse
+on:
+  workflow_dispatch:
+    inputs: 
+      version:
+        description: 'Which branch do you want to test?' # Limited to branch for now
+        required: false
+        default: 'master' 
+jobs:
+  lighthouse:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.version }}
+      - uses: actions/setup-node@v1
```

Comment:
Looks good as long as we're installing a minimum of version 10 of node.

Reply:
https://github.com/nasa/openmct/issues/3952